nncf/experimental/openvino_native/quantization/algorithms/min_max/openvino_backend.py

```diff
@@ -94,16 +93,29 @@ def _get_reduction_shape_and_use_abs_max(
         if not quantizer_config.per_channel:
             return None, use_abs_max
 
+        node = nncf_graph.get_node_by_name(target_point.target_node_name)
         if not target_point.is_weight_target_point():
-            # TODO: support reduction shapes for 3D-5D conv cases
-            return (0, 2, 3), use_abs_max
+            if target_point.type == TargetType.PRE_LAYER_OPERATION:
+                shape = nncf_graph.get_input_edges(node)[target_point.port_id].tensor_shape
+            elif target_point.type == TargetType.POST_LAYER_OPERATION:
+                shape = nncf_graph.get_output_edges(node)[target_point.port_id].tensor_shape
+            else:
+                raise NotImplementedError(f'Unsupported target point type {target_point.type}.')
+
+            channel_axis = 1
+            axes = tuple(i for i, _ in enumerate(shape) if i != channel_axis)
```

Comment:
My suggestion is to use `range(len(shape))` instead of `enumerate(shape)`. 

```python
axes = tuple(i for i in range(len(shape)) if i != channel_axis)
```

Could you please add a short comment in the code about why `channel_axis ` equals `1`?